src/DynamicData/Cache/ChangeSet.cs

```diff
@@ -5,6 +5,44 @@
 // ReSharper disable once CheckNamespace
 namespace DynamicData;
 
+/// <summary>
+/// A collection of changes with some arbitrary additional context
+/// </summary>
+/// <typeparam name="TObject">The type of the object.</typeparam>
+/// <typeparam name="TKey">The type of the key.</typeparam>
+/// <typeparam name="TContext">The additional context.</typeparam>
+public sealed class ChangeSet<TObject, TKey, TContext> : ChangeSet<TObject, TKey>, IChangeSet<TObject, TKey, TContext>
```

Comment:
I'm going to have to keep in mind this is a thing now.  I can't quite place it, but I feel like I needed it before...

Reply:
Yes, I needed some extra params to be passed down the chain, and historically I have inherited and created a new changeset. However I think this is an improvement.

Reply:
> I'm going to have to keep in mind this is a thing now. I can't quite place it, but I feel like I needed it before...

I'll wager it was for one of your various stream merging scenarios.